home.html

```diff
@@ -0,0 +1,165 @@
+                        </li>
+                        <li>
+                            <a href="">join</a>
+                        </li>
+                        <li>
+                            <a href="">sponsor</a>
+                        </li>
+                        <li>
+                            <a href="">news</a>
+                        </li>
+                        <li>
+                            <a href="">the portal!</a>
+                        </li>
+                    </ul>
+                </div>        
+            </div>
+        </div>
+        <div class="body-header">
+            <button class="btn btn-header"><i class="fa fa-bars"></i></button>
+            <div class="header-title">
+                <p class="first-text">"hello!future micronauts"</p>
+                <h1>surviving the microverse portal 2021.</h1>
+                <p class="header-desc">Have you recently been accepted into Microverse?Do you wonder what the start of your journey (The portal!) through Microverse will be?. Then, attend this forum as we'd discuss what to expect during your first few weeks as a  Micronaut and how to go through it seamlessly</p>
+                <h2>2021.11.01 (MON) ~ 02 (TUE) </h2>
+                <h3>@national staduim surulere lagos state, nigeria</h3>
+            </div>
+        </div>
+    </header>
+    <section class="program">
+        <article class="program-container">
+            <div class="program-header">
+                <h2>Main Program</h2>
+                <div class="underline"></div>
+            </div>
+            <div class="program-content-container">
+                <article class="program-content">
+                    <i class="fa fa-chalkboard-teacher"></i>
+                    <h2>lecture</h2>
+                    <p  class="center">listen to the micronauts that have just finished the portal talk about their experience</p>
+                </article>
+                <article class="program-content">
+                    <i class="fa fa-empire"></i>
+                    <h2>cc exhibition</h2>
+                    <p  class="center">get to see exciting projects you can expect to work on during the portal</p>
+                </article>
+                <article class="program-content">
+                    <i class="fas fa-comment"></i>
+                    <h2>forum</h2>
+                    <p class="center">Ask personal questions about what you want to know about the portal</p>
+                </article>
+                <article class="program-content">
+                    <i class="fa fa-superpowers"></i>
+                    <h2>workshop</h2>
+                    <p  class="center">get to see exciting projects you can expect to work on during the portal</p>
+                </article>
+                <article class="program-content p-content">
+                    <i class="fas fa-network-wired"></i>
+                    <h2>CC ignite</h2>
```

Comment:
- [ ] As pointed out by the last reviewer, please follow the design template as much as possible. Below is what this section currently looks like in your website. Please pay close attention to the background color as well as the font size. The original background color is a shade of grey.
![Screenshot 2021-10-13 at 10-44-05 Overcoming the Portal ](https://user-images.githubusercontent.com/70150423/137119153-35f2ea94-e40e-457c-b872-1d6d49a2970a.png)